azerothcore/azerothcore-wotlk
# fix(Core/SpellMgr): Essence of Wintergrasp

## Changes Proposed:
Prevents players from getting the Essence of Wintergrasp buff while Wintergrasp is disabled in worldserver.conf


## Issues Addressed:
- Closes #4109 


## How to Test the Changes:
Set `Wintergrasp.Enable=1`
Restart server, check if horde or alliance players get Essence of Wintergrasp in Dalaran.

Set `Wintergrasp.Enable=0`
Restart server, check if horde or alliance players get Essence of Wintergrasp in Dalaran.


## Target Branch(es):


## How to Test AzerothCore PRs
 
When a PR is ready to be tested, it will be marked as **[WAITING TO BE TESTED]**.

You can help by testing PRs and writing your feedback here in the PR's page on GitHub. Follow the instructions here:

http://www.azerothcore.org/wiki/How-to-test-a-PR
